namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/resources/ecr.tf

```diff
@@ -0,0 +1,72 @@
+    / software vulnerabilities in your image and the results can be viewed in the console. For further details, please see: 
+    https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html 
+    To disable 'scan_on_push', set it to false as below:
+  scan_on_push = "false"  
+  */
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "example_team_ecr_credentials" {
+  metadata {
+    name      = "example-team-ecr-credentials-output"
+    namespace = "<NAMESPACE>"
+  }
+
+  data = {
+    access_key_id     = module.example_team_ecr_credentials.access_key_id
+    secret_access_key = module.example_team_ecr_credentials.secret_access_key
+    repo_arn          = module.example_team_ecr_credentials.repo_arn
+    repo_url          = module.example_team_ecr_credentials.repo_url
+  }
+}
+
+
+/*
+#############################################SLACK NOTIFIATIONS OF ECR SCAN RESULTS########################
+To send notifications to slack of the ECR image scan results, you may insert the following lambda module that creates the slack lambda function and the event bridge. 
+The event bridge will be triggered every time there is a scan completed for your ECR repo. The event bridge executes the lambda function which then interacts with slack. A notification containing the scan result will then be sent to your slack channel as per the slack token you specify.
+The lambda function incorporates the slack token and ECR repository when it is created. The slack_token and ECR repository must be stored as a kubernetes secret, which you must create as follows:
+This secret needs to have the following two keys:
+Key 1: repo (without the prefix e.g if the url is 754256621582.dkr.ecr.eu-west-2.amazonaws.com/webops/webops-ecr1:rails, then in this case you need to supply 'webops/webops-ecr1')
+Key 2: token
+Below is a sample kubernetes secret yaml you can use to create the secret containing the slack token and ECR repo: 
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <SLACK_SECRET_NAME>
+  namespace: <NAMESPACE>
+data:
+  repo: <ECR_REPO_BASE64_ENCODED>
+  token: <SLACK_TOKEN_BASE64_ENDCODED>
+Note that the <ECR_REPO_BASE64_ENCODED> and <SLACK_TOKEN_BASE64_ENDCODED> must be encoded as base64.
+e.g 'echo -n <SLACK_TOKEN> | base64'
+As this file will contain the slack token it is important that it is encyrpted within the repo that has git-encrypt. Also the file must reside within your own team's repo and not a repo that is shared between teams such as the 'cloud-platform-environments'.
+Save the above secret yaml with the desired name and create the secret as follows: 
+kubectl create -f <SLACK_SECRET_FILE_NAME>
+Lastly, after you have created your kubernetes slack secret as above, move the following lambda module outside the comments section so that it is created alongside your ECR resource. 
+module "ecr_scan_lambda" {
```

Comment:
The closing `*/` is missing here, which is why the plan pipeline shows an error:
https://concourse.cloud-platform.service.justice.gov.uk/builds/7335

Reply:
Apologies, this file was committed in error